### ISSUE TITLE
<a name="Name"> gives errors

### ISSUE THREAD

**Issue by tbabec** [phseiff/github-flavored-markdown-to-html#55]
I have a bunch of md files which contains "<a name=...". This gives the following error during conversion:

```
Traceback (most recent call last):
  File "/home/me/content.py", line 135, in convert_pages
    htmlcontent = gh_md_to_html.main('README.md', core_converter="OFFLINE")
  File "/opt/pyenv/versions/default/lib/python3.9/site-packages/gh_md_to_html/__init__.py", line 1122, in main
    if link_location.startswith("#"):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

Looking at line 1122 makes it obvious:
```
    html_soup = BeautifulSoup(html_rendered, 'html.parser')
    for link_soup_representation in html_soup.find_all("a"):
        link_location = link_soup_representation.get("href")
        if link_location.startswith("#"):
```

It is expected that a tags exist as "<a href=" but I don't have that. I only have "<a name=" in the md-files.

I've done a workaround where I convert "<a name" to "<a href" before using gh-md-to-html.

**Comment by phseiff** [comment]
Hello, and thanks for your issue report and already finding the root of the problem. I must confess that I wasn't aware that links without a `href` are valid, and I will fix this sometime this evening (CET).

Glad you already found a workaround that works for now!

**Comment by phseiff** [comment]
This should be fixed in version `1.17.7` now, to which you can update e.g. via `pip3 install --upgrade gh-md-to-html>=1.17.7`.

Note that the names of anchors, like all id values and document-internal links will have a "`user-content-`" added before them, like this:

* `<a href="#foobar">` -> `<a href="#user-content-foobar">`
* `<a name="foobar">` -> `<a name="user-content-foobar">`
* `<a id="foobar">` -> `<a id="user-content-foobar">`
* `<p id="foobar">` -> `<p id="user-content-foobar">`

This is in accordance to GitHub's own behavior (or at least the behavior of their public API, since they changed the behavior in READMEs some time ago, or so it seems).
